apps/api/src/server/swagger-output.json

```diff
@@ -6681,6 +6701,26 @@
 					"description": "Document file name",
 					"example": "document.pdf"
 				},
+				"description": {
```

Comment:
those are duplicated - is it intentional?

Reply:
They're actually different Swagger types, the diff is just collapsed so it looks like a duplicate 😄

Reply:
ok cool!